phosphobot/pyproject.toml

```diff
@@ -83,19 +85,39 @@ openpi-client = { git = "https://github.com/phospho-app/openpi.git", subdirector
 phosphobot = "phosphobot.main:cli"
 
 [build-system]
-requires = ["setuptools>=42", "wheel", "toml"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
 
+# Hatch build configuration to include resources
+[tool.hatch.build.targets.wheel]
+packages = ["phosphobot"]
 
-[tool.setuptools]
-# include data files as listed in MANIFEST.in
-include-package-data = true
+[tool.hatch.build.targets.wheel.force-include]
+"resources" = "phosphobot/resources"
 
-[tool.setuptools.packages.find]
-where = ["."]
-include = ["*"]
+[tool.hatch.build.targets.sdist]
+include = [
+    "phosphobot/",
+    "resources/",
+    "README.md",
+    "LICENSE",
+]
```

Comment:
this won't include the dashboard in the built package. This means that you can no longer install phosphobot with pip, and this will break many users installation.